### DISCUSSION TITLE
Can we find a way to avoid having 100+ eslintrc files ?

### DISCUSSION THREAD

**Discussion by zackarydev** [nrwl/nx#19644] [Feature Requests]
Why do we need to have 1 eslintrc file per project? They're all essentially empty. I understand that they could be sometimes useful but having them in each library and app seems a bit useless...

**Comment by jaysoo** [comment]
Hi @zackarydev The way that lint targets are setup now requires these eslintrc files per-project. We are making ongoing effort to reduce configuration as much as possible, so we'll consider this case as we're making improvements in this area.

**Comment by rconjoe** [comment]
@zackarydev [Nx - Switching to ESLint's flat config format](https://nx.dev/recipes/tips-n-tricks/flat-config)

@ZackDeRose provides great insight [here](https://www.youtube.com/watch?v=32XH909CZrY) as well
